src/System.Text.Json/tests/JsonDocumentTests.cs

```diff
@@ -28,6 +28,18 @@ public static class JsonDocumentTests
         private static readonly Dictionary<TestCaseType, string> s_compactJson =
             new Dictionary<TestCaseType, string>();
 
+        private const string CompiledNewline = @"
+";
+
+        private static readonly JsonDocumentOptions s_options =
+            new JsonDocumentOptions
+            {
+                CommentHandling = JsonCommentHandling.Skip,
+            };
+
+        private static readonly bool s_replaceNewlines =
+            !StringComparer.Ordinal.Equals(CompiledNewline, Environment.NewLine);
```

Comment:
None of `CompiledNewline`, `s_options`, or `s_replaceNewlines` is used any longer, they can be removed.

Reply:
removed dead code.